### PR TITLE
[core,rdp] make monitor layout PDU optional

### PR DESCRIPTION
**Pull request by akallabeth**
the PDU might arrive or not, do not get stuck if not.

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test properly please

**Comment by akallabeth** [comment]
@freerdp-bot test

**Review by pnowack — APPROVED** [review]
Can't spot an issue here. Works fine in testing (against both MS Windows RDS and g-r-d, regardless whether the server side sends a Monitor Layout PDU here).

File: libfreerdp/core/info.c [hunk 1]
```diff
@@ -73,14 +73,18 @@ static const struct info_flags_t info_flags[] = {
 	{ INFO_HIDEF_RAIL_SUPPORTED, "INFO_HIDEF_RAIL_SUPPORTED" },
 };
 
-static BOOL rdp_read_info_null_string(const char* what, UINT32 flags, wStream* s, size_t cbLen,
-                                      CHAR** dst, size_t max, BOOL isNullTerminated)
+static BOOL rdp_read_info_null_string(rdpSettings* settings, size_t id, const char* what,
+                                      UINT32 flags, wStream* s, size_t cbLen, size_t max,
+                                      BOOL isNullTerminated)
 {
 	CHAR* ret = NULL;
```
Comment (pnowack) [inline libfreerdp/core/info.c:80]: Actually a small nit: This variable is now unused.

Comment (akallabeth) [reply libfreerdp/core/info.c:80]: true.

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**